boost-common/src/main/java/io/openliberty/boost/utils/SpringBootUtil.java

```diff
@@ -206,4 +246,96 @@ public static Properties getSpringBootServerProperties(String buildDir) throws I
+        
+        logger.info("Generating Liberty server configuration");
+        
+        // Generate Liberty configuration
+        LibertyServerConfigGenerator serverConfig = new LibertyServerConfigGenerator(libertyServerPath);
+    
+        // Find and add appropriate springBoot features
+        List<String> featuresNeededForSpringBootApp = getLibertyFeaturesForSpringBoot(springBootVersion,
+                springBootStarters, logger);
+        
+        serverConfig.addFeatures(featuresNeededForSpringBootApp);
+         // Get Spring Boot server properties
+        Properties springBootServerProps = new Properties();
+        
+        springBootServerProps = getSpringBootServerProperties(springBootProjectResources);
+        
+        // Configure SSL and endpoints
+        if (springBootServerProps.containsKey(SpringBootUtil.SERVER_SSL_KEYSTORE)) {
+            
+            Map<String, String> keystoreProperties = new HashMap<String, String>();
+            Map<String, String> keyProperties = new HashMap<String, String>();
+            
+            // For each Spring Boot keystore property, add an entry to the keystore map which maps our 
+            // Liberty keystore attribute to a bootstrap variable with the same name as the Spring Boot property.
+            // The Spring Boot properties will then be added to the server's bootstrap.properties file. 
+            keystoreProperties.put(ConfigConstants.KEYSTORE_LOCATION, "${" + SpringBootUtil.SERVER_SSL_KEYSTORE + "}");
+            
+            if (springBootServerProps.containsKey(SpringBootUtil.SERVER_SSL_KEYSTORE_PASSWORD)) {
+                keystoreProperties.put(ConfigConstants.KEYSTORE_PASSWORD, "${" + SpringBootUtil.SERVER_SSL_KEYSTORE_PASSWORD + "}");
+            }
+            if (springBootServerProps.containsKey(SpringBootUtil.SERVER_SSL_KEYSTORE_TYPE)) {
+                keystoreProperties.put(ConfigConstants.KEYSTORE_TYPE, "${" + SpringBootUtil.SERVER_SSL_KEYSTORE_TYPE + "}");
+            }
+            if (springBootServerProps.containsKey(SpringBootUtil.SERVER_SSL_KEYSTORE_PROVIDER)) {
+                keystoreProperties.put(ConfigConstants.KEYSTORE_PROVIDER, "${" + SpringBootUtil.SERVER_SSL_KEYSTORE_PROVIDER + "}");
+            }
+            
+            // Add any key properties to the separate key map.
+            if (springBootServerProps.containsKey(SpringBootUtil.SERVER_SSL_KEY_PASSWORD)) {
+                keyProperties.put(ConfigConstants.KEY_PASSWORD, "${" + SpringBootUtil.SERVER_SSL_KEY_PASSWORD + "}");
+            } 
+            if (springBootServerProps.containsKey(SpringBootUtil.SERVER_SSL_KEY_ALIAS)) {
+                keyProperties.put(ConfigConstants.KEY_NAME, "${" + SpringBootUtil.SERVER_SSL_KEY_ALIAS + "}");
```

Comment:
Since you are having to prepend `${` and append `}` for every one of these properties, why not declare constants with those already defined?

Reply:
Or, I was just thinking just have a helper method that takes in the string and wraps it in the brackets.

Reply:
But those values never change. So it really calls for constants.